File: tests/e2e/runtime/bookstore/postgresql/java/src/main/java/com/example/db/com/example/db/models/Product.java
package com.example.db.models;

public record Product(
    java.util.UUID id,
    String sku,
    String name,
    boolean active,
    Float weightKg,
    Double rating,
    java.util.List<String> tags,
    String metadata,
    byte[] thumbnail,
    java.time.LocalDateTime createdAt,
    short stockCount
) {}

File: tests/e2e/fixtures/bookstore/sqlite/kotlin/src/main/kotlin/com/example/db/com/example/db/models/Product.kt
package com.example.db.models

data class Product(
    val id: String,
    val sku: String,
    val name: String,
    val active: Int,
    val weightKg: Float?,
    val rating: Float?,
    val metadata: String?,
    val thumbnail: ByteArray?,
    val createdAt: String,
    val stockCount: Int
)
